src/mcts/params.cc

```diff
@@ -206,20 +206,18 @@ const OptionId SearchParams::kHistoryFillId{
     "synthesize them (always, never, or only at non-standard fen position)."};
 const OptionId SearchParams::kMovesLeftFactorId{
     "moves-left-factor", "MovesLeftFactor",
-    "Bonus to add to the score of a node based on how much shorter/longer "
+    "Maximum bonus to add to the score of a node based on how much shorter/longer "
```

Comment:
nit: This line exceeds 80 characters.

This should probably be renamed too. For example `MovesLeftLimit`.

Reply:
I agree that the name `MovesLeftFactor` doesn't exactly transport what it is supposed to represent, but I didn't want to obfuscate things by changing a name without changing what it is doing on my own.
What do you think about something like `MovesLeftMaxEffect`?